pillar/top.sls

```diff
@@ -4,6 +4,12 @@ base:
     - common
     - environment_settings
     - fluentd
+  'P@environment:(rc.*|.*-qa|ci)':
+    - match: compound
+    - elastic_stack.version_qa
+  'not P@environment:(rc.*|.*-qa|ci)':
```

Comment:
You can just do a regex of `.*production.*` for the environment grain.

Reply:
I guess that's not entirely accurate because of the `operations` env...

Reply:
We have the environments `ocw` and `operations` in our production VPN.

Reply:
Should I treat those separately, perhaps?

Reply:
No, it's fine to do it as a single one. The original approach of using `not` is probably easiest.

Reply:
OK. I kept it as one condition, but I removed "ci" in commit 9d2548ca.

I'll have to use care when adding Beats in `salt/top.sls` and not have conditions that are too loose and install it on the wrong minions.